src/content/developers/docs/consensus-mechanisms/index.md

```diff
@@ -42,57 +40,69 @@ A consensus mechanism in a cryptoeconomic system also helps prevent certain kind
 <!-- ### Fault tolerance -->
 <!-- TODO explain how protocols must be fault tolerant -->
 
-## Types of consensus mechanisms {#types-of-consensus-mechanisms}
+## Types of consensus protocols {#types-of-consensus-protocols}
 
 <!-- TODO -->
 <!-- Why do different consensus protocols exist? -->
 <!-- What are the tradeoffs of each? -->
 
-### Proof-of-work {#proof-of-work}
+### Nakamoto Consensus {#nakamoto-consensus}
 
-Ethereum, like Bitcoin, currently uses a **proof-of-work (PoW)** consensus protocol.
+Ethereum, like Bitcoin, currently uses the Nakamoto, or "Longest Chain", consensus protocol.
 
 #### Block creation {#pow-block-creation}
 
-Proof-of-work is done by [miners](/developers/docs/consensus-mechanisms/pow/mining/), who compete to create new blocks full of processed transactions. The winner shares the new block with the rest of the network and earns some freshly minted ETH. The race is won by whosever computer can solve a math puzzle fastest – this produces the cryptographic link between the current block and the block that went before. Solving this puzzle is the work in "proof-of-work".
+Nakamoto consensus is done by [miners](/developers/docs/consensus-mechanisms/pow/mining/) who compete to create new blocks full of processed transactions. They do this through a process known as "proof-of-work" mining (PoW). In PoW, miners are creating random numbers, called a nonce, and plugging that into a hash function to see if they produce a hash which satisfies a condition difficult enough to show they've put work into solving the problem. Once discovered, the miner who found the nonce proposes a new block to the rest of the network. In this new block, the miner pays itself in ETH. If the network decides to continue to build on his block, it becomes part of the "cannonical chain" and the miner has some freshly minted ETH. 
+
+The race is won by whosever computer can solve a math puzzle fastest – this produces the cryptographic link between the current block and the block that went before. Solving this puzzle puts the work in "proof-of-work".
```

Comment:
Isn't it supposed to be "The race is won by whichever* computer..."?

Reply:
@nassersaazi whosever is technically correct.

> Whosever: belonging to or associated with whichever person; whoever's.

But given that miners are largely server farms now (instead of home miners) I'd be good with changing this.